README-dev.md

```diff
@@ -131,100 +168,87 @@ let g:syntastic_ocaml_checkers=['merlin']
-https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt .
+Additionally, you must provide a list of peers to connect to to bootstrap the node.
+The list of peers depends on the network you want to connect to and is announced when the network is being launched. For Mainnet, the list of peers is avaialable at:
+https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt.
 
-Moreover, `daemon.json` config file also contains some bootstrap data specific to the network
-the node is trying to connect to. Therefore it also needs to be tailored specifically for
-a particular network. This file can also override some of the configuration options selected
-during compilation (see above). At the moment it can be extracted from the docker image
-dedicated to running a particular network. If it's not located in the config directory,
-it can be pointed to with `--config-file` option.
+The `daemon.json` config file also contains bootstrap data that is specific to the network the node is trying to connect to and must be tailored specifically for a particular network. This file can also override some of the configuration options selected during compilation. The `daemon.json` file can be extracted from the Docker image
+that is dedicated to running a particular network. If it's not located in the `config` directory, it can be pointed to with `--config-file` option.
 
-When all of this setup is complete, the daemon can be launched (assuming the key passphrase was set to "pass"):
+When all of this setup is complete, you can launch the daemon. The following command assumes the key passphrase is set to `pass`:
 
 ```shell
 $ MINA_LIBP2P_PASS=pass dune exec src/app/cli/src/mina.exe -- daemon --libp2p-keypair /path/to/key --peer-list-url https://example.peer.list --config-file /custom/path/to/daemon.json
 ```
 
-The `--seed` flag tells the daemon to run a fresh network of its own. In that case specifying
-a peer list is not necessary, but still possible. With `--seed` option the node will
-not crash, even if it does not manage to connect to any peers. See:
+The `--seed` flag tells the daemon to run a fresh network of its own. When this flag is used, specifying a peer list is not required, but is still possible. With `--seed` option the node does not crash, even if it does not manage to connect to any peers. To learn more, see the command line help:
 
 ```shell
 $ dune exec src/app/cli/src/mina.exe -- -help
 ```
 
-to learn about other options to the CLI app and how to connect to an existing network, such
-as mainnet.
+The command line help is the place to learn about other options to the Mina zkApp CLI and how to connect to an existing network, such as Mainnet.
 
 ## Using the Makefile
 
-The makefile contains phony targets for all the common tasks that need to be done.
-It also knows how to use Docker automatically. 
+The makefile contains placeholder targets for all the common tasks that need to be done and automatically knows how to use Docker. 
```

Comment:
```suggestion
The Makefile contains placeholder targets for all the common tasks that need to be done and automatically knows how to use Docker. 
```
Although in principle the makefile  can be named anyhow, it is customary to name if `Makefile` (with capital "M"). This also the name used in Mina, so I'd use the capital letter to avoid confusion.